tao/tao_bridge/kernels/launch_base.cc

```diff
@@ -125,6 +125,12 @@ std::vector<int> LaunchBase::ResourcesVector() {
     num_fixed_shaped = fixed_shaped_types.size();
```

Comment:
can polish code as the following, and remove the `num_fixed_shape` variable.
``` c++
DataTypeVector fixed_shaped_types;
if (ctx_->HasAttr("Tfixedshapes"))  {
    OP_REQUIRES_OK_RETURN(ctx_, std::vector<int>(),
                          ctx_->GetAttr("Tfixedshapes", &fixed_shaped_types));
}
```

Reply:
done